adobe/spectrum-css
# fix(clear button): align background color and hover state for disabled variant

## Description

More closely aligns the clear button styling with the close button styling by adopting the pattern of classes targeting the disabled state. Scopes hover indicators to components that are not disabled and aligns the disabled colors used by the two components.

## How and where has this been tested?

Locally in storybook.

### Validation steps

1. Fetch the branch and run it locally or access the Storybook URL for the branch.
2. Navigate to the clear button component and toggle on the disabled state.
3. Do the same for the close button.
4. Verify that the two are visually consistent.

### Regression testing

Validate:

1. The documentation pages for at least two other components are still loading, including:


5. If components have been modified, VRTs have been run on this branch:


## Screenshots

![Screenshot 2024-10-15 at 10 39 50 AM](https://github.com/user-attachments/assets/c525b9a8-a6a9-4046-ab8f-af25fd26f66d)

## To-do list
